itests/util/src/main/java/org/apache/hadoop/hive/cli/control/CliConfigs.java

```diff
@@ -348,13 +348,13 @@ public CompareCliConfig() {
     }
   }
 
-  public static class NegativeCliConfig extends AbstractCliConfig {
-    public NegativeCliConfig() {
+  public static class NegativeLlapLocalCliConfig extends AbstractCliConfig {
```

Comment:
I did not understand your point. I just renamed the old drivers. I had to rename NegativeMinimr because it has "mr" in it. I admittedly did not have to rename NegativeCliDriver. I changed that one to make the naming consistent with other drivers (MiniLlapCliDriver and MiniLlapLocalCliDriver). Also, all 4 llap drivers use the same config directory. It is easier to identify they all use the same runtime with similar names.

Reply:
after this patch we will have the following llap tests:

* TestMiniLlapLocalCliDriver
* TestNegativeLlapLocalCliDriver
* TestMiniLlapCliDriver
* TestNegativeLlapCliDriver

I'm just saying that instead of adding keywords to the test name we could remove some...nowadays `TestMiniLlapLocalCliDriver` means the same to me as what `TestCliDriver` have been a few years ago

Another interesting thing is the presence of the "Mini" in the name of some of the llap drivers....